data/moves.js

```diff
@@ -8530,7 +8530,7 @@ exports.BattleMovedex = {
 		basePower: 0,
 		category: "Status",
 		desc: "The user has 1/16 of its maximum HP restored at the end of each turn, but it is prevented from switching out and other Pokemon cannot force the user to switch out. The user can still switch out if it uses Baton Pass, Parting Shot, U-turn, or Volt Switch. If the user leaves the field using Baton Pass, the replacement will remain trapped and still receive the healing effect. During the effect, the user can be hit normally by Ground-type attacks and be affected by Spikes, Toxic Spikes, and Sticky Web, even if the user is a Flying type or has the Ability Levitate.",
-		shortDesc: "User recovers 1/16 max HP per turn. Traps user.",
+		shortDesc: "Traps/grounds user; heals 1/16 max HP per turn.",
```

Comment:
...I would rather the healing effect be first, considering it's more important.

`Heals 1/16 max HP per turn; traps/grounds user`

Reply:
@Marty-D, thoughts?

Reply:
Yeah, that's fair. I reversed them so it might be clearer both effects are for the user, but I guess it doesn't matter. Plus H/t is thinner than T/h in Verdana so win-win. Will switch them back in my next desc update.